spec/javascripts/helpers/jasmine-jquery.js

```diff
@@ -0,0 +1,841 @@
+/*!
```

Comment:
Yes - it's used to power the jQuery assertions like `expect($foo).toHaveClass('bar')`. The JS test dependencies are just checked in - we're currently not using a package manager to add them.